lib/panda/client.rb

```diff
@@ -52,6 +52,18 @@ def report(advertiser_id, report_type, dimensions, params = {})
       )
     end
 
+    def token_info(app_id)
+      get_token('tt_user/token_info/get/', app_id: app_id, access_token: access_token)
+    end
+
+    private
+
+    def get_token(path, params = {})
+      request = Panda::HTTPRequest.new('GET', path, params)
+      response = Panda.make_get_request(request)
+      response.status == 200 ? response.parsed_body : nil
```

Comment:
There is no need in checking status, error middleware will handle all non successful requests. Please add a separate structure that wraps raw response from TikTok like we do with collections

Reply:
Collection is a wrapper around enumerable with pagination. We don't have pagination here, I can just fetch `data` from response

Reply:
We wrap raw responses from API with our objects. Please, create a response object for this API and return it from this method